client/src/components/GeneratedLink.js

```diff
@@ -16,8 +17,12 @@ const GeneratedLink = () => {
         <Avatar src="/photos/logo2.png" alt="Avatar" />
         <Text>
           <Title>{title}</Title>
-          <ShortLink>{shortUrl}</ShortLink>
-          <LongLink>{longUrl}</LongLink>
+          <ShortLink href={realUrl} target="_blank" rel="noopener noreferrer">
```

Comment:
Please don't do the front-end re-direct to the long url, the redirect should happen in backend